pkg/backup/http.go

```diff
@@ -92,6 +96,10 @@ func (b *Backup) serveSnap(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	w.Header().Set(HTTPHeaderEtcdVersion, "v"+getVersionFromBackup(fname))
```

Comment:
Just to make input and output consistent.

After a second thought I think we should remove the "v" in input. The user docs doesn't have "v".